common/index.php

```diff
@@ -35,6 +35,26 @@
 				hash = "/index"
 			}
 			navigate(hash)
```

Comment:
分号还是要加吧？
```suggestion
			navigate(hash);
```
上面和下面的也是。

Reply:
确实